css/css-text-decor/text-combine-emphasis.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>CSS Text: U+FFFC OBJECT REPLACEMENT CHARACTER presence</title>
+<link rel="author" title="Myles C. Maxfield" href="mailto:mmaxfield@apple.com">
+<link rel="help" href="https://www.w3.org/TR/css-text-3">
```

Comment:
Extreme nit: the canonical URL has a slash at the end.

Reply:
we also often refer to the `drafts.csswg.org` URL